src/settings/validateNameAndCode.test.js

```diff
@@ -0,0 +1,19 @@
+import '../../test/jest/__mock__';
+import validateNameAndCode from './validateNameAndCode';
+
+describe('ValidateNameAndCode', () => {
+  it('should return errors when code & name are empty', () => {
+    const item = {};
+    const errors = validateNameAndCode(item);
+    expect(JSON.stringify(errors.code)).toMatch(/ui-inventory.fillIn/i);
+    expect(JSON.stringify(errors.name)).toMatch(/ui-inventory.fillIn/i);
```

Comment:
You may find it useful to [mock react-intl as in ui-users](https://github.com/folio-org/ui-users/blob/f96b1243f0b0a576b29d37d8cb742d455e9a6d13/test/jest/__mock__/intl.mock.js) so you can write tests that a little bit more concise:
```
expect(errors.code).toBe("ui-inventory.fillIn");
```

Reply:
We tried in this way but was getting error. So we kept as previous to match.